Jetcaster/testing/build.gradle.kts

```diff
@@ -0,0 +1,45 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+}
+
+android {
+    namespace = "com.example.testing"
```

Comment:
Namespace should be "com.example.jetcaster.core.data.testing"

Reply:
done